ckan/config/middleware/flask_app.py

```diff
@@ -60,6 +61,19 @@
 
 log = logging.getLogger(__name__)
 
+csrf = CSRFProtect()
+
+csrf_warn_extensions = (
+        "Extensions are excluded from CSRF protection! "
+        "We allow extensions to run without CSRF protection "
+        "but will be forced in the future. "
+        "To be able to use CSRF protection you would need to "
+        "set ckan.csrf_protection.ignore_extensions=False in your .ini file "
+        "and to set the csrf_token in your forms. "
+        "e.g {% import 'macros/form.html' as form %} "
+        "then you can use it like: {{ form.csrf_protection() }}"
+    )
```

Comment:
```suggestion
csrf_warn_extensions = (
        "Extensions are excluded from CSRF protection! "
        "We allow extensions to run without CSRF protection "
        "but it will be forced future releases. "
        "Read the documentation for more information on how to add "
        "CSRF protection to your extension."        
    )
```